pkg/projector/slide/current_speaker_chyron.go

```diff
@@ -87,11 +105,24 @@ func CurrentSpeakerChyronSlideHandler(ctx context.Context, req *projectionReques
 		}
 	}
 
-	// TODO: Also include agenda item number and number
-	// coTitle := los.ContentObject().Get("title")
-	// if coTitle != nil {
-	// agendaItem = coTitle.(string)
-	// }
+	co, err := viewmodels.GetTitleInformationByContentObject(ctx, req.Fetch, los.ContentObjectID)
+	if err != nil {
+		return nil, fmt.Errorf("could not fetch coTitle information")
+	}
+	switch co.Collection {
+	case "motion", "topic":
+		slideAgendaItem = strings.Join(func() []string {
+			parts := []string{}
+			for _, s := range []string{co.AgendaItemNumber, co.Number, co.Title} {
+				if strings.TrimSpace(s) != "" {
+					parts = append(parts, s)
+				}
+			}
+			return parts
+		}(), " · ")
```

Comment:
Wrapping this into a `func` is a bit weird. You could also use `slices.Collect` to construct the parts slice.